src/driver/cockroachdb/CockroachQueryRunner.ts

```diff
@@ -122,6 +122,10 @@ export class CockroachQueryRunner
      * You cannot use query runner methods once its released.
      */
     release(): Promise<void> {
+        if (this.isReleased) {
```

Comment:
what is the motivation behind this change? what was the problem caused these lines to be added?